File: 7KYU/Get the MIddle Character/solution.clj
(ns getmiddle.core)

(defn get-middle [word]
  (let [middle-index (Math/floor (/ (count word) 2))]
    (if (even? (count word))
      (str (nth word (dec middle-index)) (nth word middle-index))
      (str (nth word middle-index)))))

File: 7KYU/Get the MIddle Character/solution.py
import math
def get_middle(s):
    length = len(s)
    if length % 2 != 0:
        return str(s[math.floor(length / 2)])
    else:
        return str(s[math.floor(length / 2)-1]) + str(s[math.floor(length / 2)])